swift/ql/lib/codeql/swift/generated/GetImmediateParent.qll

```diff
@@ -24,6 +24,18 @@ Element getAnImmediateChild(Element e) {
       or
       abstract_storage_decl_accessor_decls(e, _, x)
       or
+      enum_case_decl_elements(e, _, x)
```

Comment:
I assume some new kind of sorted was done here?

Reply:
ah, yes, classes are sorted by (dir, name) which bunches together same kinds (decls, expr, etc), previously they were sorted only by name